Index.html

```diff
@@ -213,17 +223,23 @@ <h1>About me</h1>
 </ul>
 </section>
 <footer class="aboutContact">
+  <div class="formResponsive">
   <p>
     I'm alwyas ineterested in hearing about new projects, so if you'd 
     like to chat please get in touch.
   </p>
   <form action="https://formspree.io/f/mvodrqbw" method="POST" class="contactForm">
     <input type="text" name="user_name" placeholder="Full Name" maxlength="30" required>
     <input type="email" name="user_email" placeholder="Email address" required>
-    <textarea name="user_text" maxlength="500"  required>Enter text here</textarea>
+    <textarea name="user_text" maxlength="500" placeholder="Enter the text here"  required></textarea>
     <button type="submit" class="contactBtn">Get in Touch</button>
   </form>
+</div>
+<div class="line">
+<hr class="responsiveHr">
+</div>
```

Comment:
- [ ] This `hr` tag is not visible on your page. The grey line should be at the top of the social media icons.

Yours | Original
----- | -----
![image](https://user-images.githubusercontent.com/16058470/122400607-c77d5300-cf73-11eb-83d3-d7def5c45cd3.png) | ![image](https://user-images.githubusercontent.com/16058470/122402498-71a9aa80-cf75-11eb-8be0-b3aeccefe97e.png)

Reply:
This `hr` still is not visible work on that and you will b good to go

Reply:
Thanks for your review. But on my side line is displaying
![ss8](https://user-images.githubusercontent.com/72971752/122549222-8a2bca80-d032-11eb-8cd6-18f9c33dd5a8.png)

Reply:
Oooh sorry I made a mistake I checked social media icons for the main section
ask another CR your project will be approved

I apologize for the inconvenience!